### PR TITLE
Use a valid app ID for PCS in e2e test

### PR DESCRIPTION
**Pull request by mikkeloscar**
Use a valid appID for PCS e2e test:

image

### CODE REVIEW

**Comment by mikkeloscar** [comment]
:+1:

File: e2e/apply/sample.yaml [hunk 1]
```diff
@@ -38,9 +38,6 @@ spec:
         - type: Ingress
           average: 20000m
       podTemplate:
-        metadata:
-          labels:
-            application: "e2e-deploy-sample"
```
Comment (demonCoder95) [inline e2e/apply/sample.yaml:43]: Is it not important to have the `application: kubernetes` label here as well?

Comment (demonCoder95) [reply e2e/apply/sample.yaml:43]: Ah, because of the parenthood. Got it. Thanks.

**Comment by linki** [comment]
👍

**Pull request merged**